### PR TITLE
Override keepAlive time to be lower then NLB idle time (350s)

### PR DESCRIPTION
**Pull request by patduin**
See for instance this blogpost: https://paramount.tech/blog/2021/07/26/mitigation-of-connection-reset-in-aws.html and https://medium.com/tenable-techblog/lessons-from-aws-nlb-timeouts-5028a8f65dda
We've seen a reduced number of 10 mins (connection timeout) reported by waggle dance once we set this lower TCP keepalive settings. Going from 30-40 timeout calls an hour to single digit. Occurrences still happen we suspect because the server side (HMS) should also set similar TCP KeepAlive settings.

### CODE REVIEW

File: templates/waggledance.json [hunk 1]
```diff
@@ -70,6 +70,20 @@
 		"softLimit": 65536,
 		"hardLimit": 65536
 	  }
-	]
+	],
+        "systemControls": [
+          {
+                "namespace": "net.ipv4.tcp_keepalive_time", 
+                "value": "200"
```
Comment (javsanbel2) [inline templates/waggledance.json:77]: The best would be to inject variables here so anyone can override based on their infrastructure.

Comment (abhimanyugupta07) [reply templates/waggledance.json:77]: will there be a separate PR for K8S - https://github.com/ExpediaGroup/apiary-federation/blob/master/k8s.tf ?

Comment (patduin) [reply templates/waggledance.json:77]: You cannot do this easily in Kubernetes, it's not allowed to set these flags.
You get errors like:
```
PodSecurityPolicy: unable
--
to admit pod: [pod.spec.securityContext.sysctls[0]: Forbidden: unsafe
sysctl "net.ipv4.tcp_keepalive_time" is not allowed
pod.spec.securityContext.sysctls[1]: Forbidden: unsafe sysctl
"net.ipv4.tcp_keepalive_intvl" is not allowed
pod.spec.securityContext.sysctls[2]: Forbidden: unsafe sysctl
"net.ipv4.tcp_keepalive_probes" is not allowed]
```

So we need to sort this out separately

File: variables.tf [hunk 2]
```diff
@@ -359,3 +359,21 @@ variable "datadog_metrics_enabled" {
   type        = bool
   default     = false
 }
+
+variable "tcp_keepalive_time" {
```
Comment (abhimanyugupta07) [inline variables.tf:363]: variables need to be added in README

Comment (patduin) [reply variables.tf:363]: yup added now

**Review by abhimanyugupta07 — APPROVED** [review]

**Pull request merged**